src/app/api/graphql/modules/transactions/graphql/transaction-create.input.ts

```diff
@@ -25,8 +17,9 @@ export class CreateTransactionDto {
   description: string;
 
   @IsNotEmpty()
-  @IsNumber({ maxDecimalPlaces: 2 })
+  @IsNumber({maxDecimalPlaces: 2})
   @Min(0)
+  //tf?
   @Max(999999999999.99)
```

Comment:
Idk what max value to use, but it should be a high value anyway and be consistent everywhere

maybe Number.MAX_SAFE_INTEGER

also check out https://v2.dinerojs.com/docs (or another equivalent) for expressing monetary values, will probably be useful later

Reply:
But is the `@Max` decorator even needed? 
I mean I don't think a transaction of 999999999999$ will ever take place

Reply:
would probably be an edge case, if parsing a file goes wrong

also, if users will be allowed to manually add transactions in the future